components/eam/src/chemistry/modal_aero/aero_model.F90

```diff
@@ -1593,7 +1593,11 @@ subroutine aero_model_emissions( state, cam_in )
        sflx(:)=0._r8
        F_eff(:)=0._r8
```

Comment:
I guess this may not be needed, since it is a local variable that is no longer being used here.

Reply:
removed F_eff